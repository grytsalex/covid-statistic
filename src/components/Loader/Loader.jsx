import React from "react";
import { SpinnerWrapper } from "./styledComponent";
import Spinner from "../../assets/spinner.svg";

export const Loader = () => {
  return (
    <SpinnerWrapper>
      <img src={Spinner} alt="no image"/>
    </SpinnerWrapper>
  );
};
