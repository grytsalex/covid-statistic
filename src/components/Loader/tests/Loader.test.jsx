import React from "react";
import { Loader } from "../Loader";
import { shallow } from "enzyme";

describe("Loader", () => {
  it("should match snapshot", () => {
    expect(shallow(<Loader />)).toMatchSnapshot();
  });
});
