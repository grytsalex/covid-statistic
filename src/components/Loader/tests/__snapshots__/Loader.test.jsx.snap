// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match snapshot 1`] = `
<styled.div>
  <img
    alt="no image"
    src="test-file-stub"
  />
</styled.div>
`;
